.github/workflows/release_pytorch_wheels.yml

```
@@ -7,14 +7,25 @@ permissions:
   contents: read
 
 jobs:
-  build_pytorch_grid:
+  setup:
+    runs-on: ubuntu-latest
```

nit: pin runner images

```suggestion
    runs-on: ubuntu-24.04
```

Generally, search around the repository for code patterns to follow: https://github.com/ROCm/TheRock/blob/9e23efdd56ab1d3f09daa1972ec2149d775a53df/.github/workflows/setup.yml#L19-L21

There are many gotchas across other projects on github that we've deliberately avoided in other workflows already.